pkg/vpa/vpa.go

```diff
@@ -258,3 +264,20 @@ func (r Reconciler) createVPA(namespace string, vpaName string) error {
 	}
 	return nil
 }
+
+func vpaUpdateModeForNamespace(ns *corev1.Namespace) v1beta2.UpdateMode {
```

Comment:
Am I reading this correctly that the updateMode is only configurable for a namespace? That's fine with me and expected based on the current configuration. I just wanted to check.

Reply:
Yes, like the other Goldilocks' managed labels on the Namespace we're sticking with the label == affects namespace pattern.